### DISCUSSION TITLE
Ability to clear/invalidate individual record from cache table of rabbitmq-auth-backend-cache

### DISCUSSION THREAD

**Discussion by mamprogr** [rabbitmq/rabbitmq-server#7715] [Other]
### Is your feature request related to a problem? Please describe.

rabbitmq-auth-backend-cache provides good and simple way to cache http response for access control operations.

The expiration of cache is currently time-based. I had a requirement in which I wanted to clear cache result for a individual user entry even before its cache expiration time. I had a look at cache code and found there is delete option for a key in rabbit_auth_cache_ets.erl (default cache implementation). The delete option is only exercised before performing put operation.

I think it will be good if we can also exposed the delete option so it can be called externally..

### Describe the solution you'd like

After research I see this command but it says "ok" but without deleting the cache..
`rabbitmqctl eval 'Key = {user_login_authentication,[<<"username">>,[{password,<<"password">>}]]}, rabbit_auth_cache_ets:delete(Key).' `

**Comment by michaelklishin** [comment]
Since plugins can provide CLI commands, a CLI command such as `clear_auth_backend_cache` that clears the entire cache would be the easiest solution to introduce.

You are welcome to submit a PR.

**Reply by michaelklishin** [reply]
Also, we cannot just assume that everyone uses `rabbit_auth_cache_ets` as there are several cache implementations.
